src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -2,25 +2,70 @@ import React from 'react';
 import styled from 'react-emotion';
 import {browserHistory} from 'react-router';
 import PropTypes from 'prop-types';
+import {omit} from 'lodash';
 
+import SentryTypes from 'app/sentryTypes';
 import AsyncComponent from 'app/components/asyncComponent';
 import InlineSvg from 'app/components/inlineSvg';
 import withApi from 'app/utils/withApi';
 import space from 'app/styles/space';
 
 import EventModalContent from './eventModalContent';
+import {getQuery} from './utils';
+
+const slugValidator = function(props, propName, componentName) {
+  const value = props[propName];
+  if (value && !/^(?:[^:]+:)?(?:[^:]+):(?:[a-f0-9]+|latest|oldest)$/.test(value)) {
```

Comment:
:nice:regex:

It'd help to have a comment with what a valid slug looks like. Are special characters (non alphanumerics) allowed in the first two groups?